### PR TITLE
Do not restore signal mask when pselect6 called without signals to block

### PR DESCRIPTION
**Pull request by dcolascione**

### CODE REVIEW

File: src/test/pselect.c [hunk 1]
```diff
@@ -0,0 +1,8 @@
+/* -*- Mode: C; tab-width: 8; c-basic-offset: 2; indent-tabs-mode: nil; -*- */
+
+#include "rrutil.h"
+
+int main(void) {
+  pselect(0, NULL, NULL, NULL, NULL, NULL);
```
Comment (rocallahan) [inline src/test/pselect.c]: It looks like this pselect call should indeed block indefinitely, as the test timeout on the buildbot suggests...

Comment (dcolascione) [inline src/test/pselect.c]: Thanks. That's what I get before I sent a diff before waiting for the tests to complete :-P

**Comment by rocallahan** [comment]
Are you going to push an updated test here?

**Comment by dcolascione** [comment]
Yes :-)

**Comment by khuey** [comment]
Superseded by #1928.

**Pull request closed without merging**